src/main.py

```diff
@@ -16,6 +16,8 @@
 from src.utils.config import TransformerConfig, TrainingConfig, VTRConfig
 from src.utils.train import train
 
+import pickle
```

Comment:
Please, use correct order of imports 😅 PyCharm can do it for you: [link](https://www.jetbrains.com/pycharm/guide/tips/optimize-imports/)